### PR TITLE
Split some sections into subsections

### PR DESCRIPTION
**Pull request by AntonioND**

### CODE REVIEW

**Comment by AntonioND** [comment]
What do you guys think of something like this?

**Review by GValiente — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -91,30 +91,66 @@ Contribute adding resources or providing feedback through Pull Requests, Issues
...
-- [gbsenpai](https://github.com/asiekierka/gbsenpai) - GB Studio player GBA port with some enhancements.
+1. C and C++
+
+   - [gba-modern](https://github.com/JoaoBaptMG/gba-modern) - Write GBA games using modern C++.
+   - [dusk](https://github.com/redthing1/dusk) - Simple, lightweight, intuitive framework for GBA game development
+   - [Butano](https://github.com/GValiente/butano) - Modern C++ high level engine for the GBA.
+
+1. D
+
+   - [GBA Dlang](https://github.com/redthing1/gba_dlang) - WIP SDK for GBA development using D, using LLVM toolchain. provides fully functional TONC, GBFS, Maxmod.
```
Comment (avivace) [inline README.md:102]: I don't think creating sublists of 1 (even 2) elements helps the visual clutter at all :/

File: README.md [hunk 2]
```diff
@@ -91,30 +91,66 @@ Contribute adding resources or providing feedback through Pull Requests, Issues
...
+   1. Software mixing
+
+      - [Apex Audio System](https://github.com/stuij/apex-audio-system) - Music and sound library (supports .mod only, but _very_ good performance)
+      - [Krawall](https://github.com/sebknzl/krawall) - Music and sound library (supports .xm, .s3m)
+      - [Maxmod](https://maxmod.devkitpro.org) - Music and sound library (supports .mod, .xm, .s3m, .it)
+      - [Pimpmobile](https://github.com/kusma/pimpmobile) - Music library (supports .mod, .xm)
+
+   1. DMG audio channels
+
+      - [GBT Player](https://github.com/AntonioND/gbt-player) - Music library that uses the DMG sound channels (close to 0% CPU usage).
```
Comment (avivace) [inline README.md:137]: Maybe in these cases we can just put in bold the main feature? e.g. here **DMG sound channels**

**Review by avivace — CHANGES_REQUESTED** [review]
Looks good, I definitely approve the biggest sublistings.. but I still wouldn't have them for 1-2 elements "groups"

File: README.md (hunk 1 above)
Comment (AntonioND) [reply README.md:102]: We have been talking about this on Discord. This is another suggestion about how to organize it: https://gist.github.com/exelotl/ae81b7906a24159008720ca37141885d

The main problem is that there isn't a clear separation between compilers, toolkits and libraries (specially seeing as some things such as Natu have everything in one!).

**Comment by AntonioND** [comment]
I'll close this until we can figure out how to display this properly.

**Pull request closed without merging**